Classes/New Issue/NewIssueTableViewController.swift

```diff
@@ -197,6 +202,15 @@ final class NewIssueTableViewController: UITableViewController, UITextFieldDeleg
             IssueTextActionOperation(icon: UIImage(named: "bar-ul"), operation: .line("- ")),
             IssueTextActionOperation(icon: UIImage(named: "bar-indent"), operation: .line("  ")),
             IssueTextActionOperation(icon: UIImage(named: "bar-link"), operation: .wrap("[", "](\(UITextView.cursorToken))")),
+            IssueTextActionOperation(icon: UIImage(named: "cloud-upload"), operation: .execute({ [weak self] in
+                guard let strongSelf = self else { return }
+                
+                let imagePicker = UIImagePickerController()
+                imagePicker.delegate = strongSelf
+                imagePicker.modalPresentationStyle = .formSheet
```

Comment:
@Sherlouk One quick suggestion here for iPad,
New issue view controller is already presented as form sheet, so if we present image picker also a form sheet it might not look good.

Also apple [suggests](https://developer.apple.com/documentation/uikit/uiimagepickercontroller) to use `UIImagePicketController` as `popover` if we target iPad

Reply:
Will investigate further, it looked fine when I was testing but didn't get `popover` an attempt

Reply:
Updated to use popover, honestly I think it looks worse than form sheet because there isn't anything visible to attach it to. It's also too small.

Still think formsheet looks better, easy switch back though